simulators/optimism/devnet/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# This simulation runs JSON-RPC API tests.
+FROM golang:1-alpine as builder
```

Comment:
Specify the exact version, e.g. `go1.18-alpine3.15`

Reply:
Yep.